Objects/Converters/ConverterCSI/ConverterCSIShared/ConverterCSI.cs

```diff
@@ -153,89 +152,82 @@ public bool CanConvertToSpeckle(object @object)
+          convertedNames = StoriesToNative(o);
           break;
-        //case CSIWindLoadingFace o:
-        //  LoadFaceToNative(o, ref appObj);
-        //  break;
-        //case CSITendonProperty o:
+        // case CSIWindLoadingFace o:
+        //   convertedName = LoadFaceToNative(o, appObj.Log);
+        //   break;
+        // case CSITendonProperty o:
         case OSG.Element1D o:
-          FrameToNative(o, ref appObj);
+          FrameToNative(o, appObj);
           break;
         case OSG.Element2D o:
-          AreaToNative(o, ref appObj);
+          AreaToNative(o, appObj);
           break;
         case LoadBeam o:
-          LoadFrameToNative(o, ref appObj);
+          convertedNames = LoadFrameToNative(o, appObj.Log);
           break;
         case LoadFace o:
-          LoadFaceToNative(o, ref appObj);
+          convertedName = LoadFaceToNative(o, appObj.Log);
           break;
-        //case osg.node o:
-        //    return pointtonative(o);
         case Geometry.Line o:
-          LineToNative(o, ref appObj); // do we really want to assume any line is a frame object?
+          convertedName = LineToNative(o); // do we really want to assume any line is a frame object?
           break;
         case OSG.Node o:
-          PointToNative(o, ref appObj);
+          convertedName = PointToNative(o, appObj.Log);
           break;
         case Property1D o:
-          Property1DToNative(o, ref appObj);
+          Property1DToNative(o, appObj);
           break;
         case StructuralMaterial o:
-          // Material conversion does not return an app object
-          MaterialToNative(o);
+          convertedName = MaterialToNative(o);
           break;
-        #region BuiltElements
         case BuiltElements.Beam o:
-          CurveBasedElementToNative(o, o.baseLine, ref appObj);
+          CurveBasedElementToNative(o, o.baseLine, appObj);
           break;
         case BuiltElements.Brace o:
-          CurveBasedElementToNative(o, o.baseLine, ref appObj);
+          CurveBasedElementToNative(o, o.baseLine, appObj);
           break;
         case BuiltElements.Column o:
-          CurveBasedElementToNative(o, o.baseLine, ref appObj);
+          CurveBasedElementToNative(o, o.baseLine, appObj);
           break;
         case GridLine o:
           GridLineToNative(o);
           break;
-        #endregion
         default:
-          appObj.Update(
-            status: ApplicationObject.State.Skipped,
-            logItem: $"Skipped not supported type: {@object.GetType()}"
-          );
-          break;
+          throw new ConversionSkippedException($"{@object.GetType()} is an unsupported type");
       }
 
-      // log
-      var reportObj = Report.ReportObjects.ContainsKey(@object.id) ? Report.ReportObjects[@object.id] : null;
-      if (reportObj != null && notes.Count > 0)
-        reportObj.Update(log: notes);
+      if (convertedName is not null)
+        convertedNames.Add(convertedName);
```

Comment:
our coding standards are to always use braces, even for one-liners. This is consistant throughout the PR,